### PR TITLE
Added Name Change

### PR DESCRIPTION
**Pull request by monikLover**
Player will now be able to change their name to whatever they want to in-game. Also has an event that will introduce the player to it and ask them whether or not they want their name changed.

### CODE REVIEW

File: Monika After Story/game/script-story-events.rpy [hunk 1]
```diff
@@ -40,3 +40,46 @@ label gender:
...
+        m "You're using '[currentuser]' and '[player]'."
+        m "Either that or you must really like that pseudonym."
+    m "Do you want me to call you something else?"
+    menu:
+        "Yes":
+            $ done = False
+            m "Ok, just type 'Nevermind' if you change your mind, [player]."
+            while not done:
+                #Could add an elif that takes off special characters
+                $ tempname = renpy.input("Tell me what is it.")
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-story-events.rpy]: You could simplify the if on the next line by calling `lower()` on the input:
`tempname = renpy.input("...").lower()`

and the if on the next line would only need to check for `"nevermind"`.

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -3483,6 +3483,56 @@ label monika_penname:
...
+        m "That's good to hear."
+    m "You want to change your name?"
+    menu:
+        "Yes":
+            m "Just type 'Nevermind' if you change your mind."
+            $ done = False
+            while not done:
+                $ tempname = renpy.input("What do you want me to call you?")
+                if tempname == "Nevermind" or tempname == "nevermind":
+                    if persistent.said_no == True:
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: no need to check if `== True`. Simplify this to:
`if persistent.said_no:`

File: Monika After Story/game/script-topics.rpy [hunk 3]
```diff
@@ -3483,6 +3483,56 @@ label monika_penname:
...
+    if persistent.said_no == True:
+        m "Have you changed your mind, [player]?"
+        m "That's good to hear."
+    m "You want to change your name?"
+    menu:
+        "Yes":
+            m "Just type 'Nevermind' if you change your mind."
+            $ done = False
+            while not done:
+                $ tempname = renpy.input("What do you want me to call you?")
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Refer to what I said in `script-story-events.rpy`

File: Monika After Story/game/script-story-events.rpy (hunk 1 above)
Comment (monikLover) [reply Monika After Story/game/script-story-events.rpy]: Null

File: Monika After Story/game/script-topics.rpy (hunk 3 above)
Comment (monikLover) [reply Monika After Story/game/script-topics.rpy]: Null

**Comment by Winblows2** [comment]
Disregard the above request. Merged.

**Pull request merged**